### PR TITLE
Feature/endpoint service by

### PR DESCRIPTION
**Pull request by JSBHA**
### 🎯 Milestone 

The endpoints of adding a new service and calling a service by id are created, the routes and the folder that will make the connection to the database are created.  

### 🛠 Changes

feat: A file is created in the controller folder called SelectServiceController.js which has the endpoint to only call one service per id 

feat:  A file is created in the controller folder called addServiceController.js which has the endpoint to create a new service  

feat: A folder called data is created containing a ServicesData.js file that contains the data and will make the connection to supabase. 

refactor: in the index.js the routers are added to add service and a response container is created for the endpoint of addServiceController.js

refactor:In the view folder ServiceView.js is updated with the controller paths of addServiceController.js and SelectServiceController.js a response container is created for all endpoints. 

### 📘 Documentation 

- [ ] I have added or updated the documentation
- [x] No documentation update is required

### CODE REVIEW

**Review by JuanP0329 — APPROVED** [review]

**Review by JuanP0329 — APPROVED** [review]

File: Controller/SelectServiceController.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// endpoint para consumir un servicio
```
Comment (FelipeVelasquez1802) [inline Controller/SelectServiceController.js]: Please, delete this comment!

File: View/ServiceView.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  }
+});
+
+
+module.exports = { ServiceRoutes};
+
+
+
+
+
```
Comment (FelipeVelasquez1802) [inline View/ServiceView.js]: Please, delete this lines

File: data/ServicesData.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      "Servicio de arreglo de barba",
+      "url_de_la_imagen",
+      1
+    ),
+  ];
+
+  return servicesMock;
+}
+
+// //se creas const servicesMock para poder add el servicio
```
Comment (FelipeVelasquez1802) [inline data/ServicesData.js]: Please, delete this comment!

File: package.json
Comment (FelipeVelasquez1802) [inline package.json:1]: This file should not be deleted!

**Review by FelipeVelasquez1802 — CHANGES_REQUESTED** [review]
# ⚠️Request changes

@JSBHA please check out this comments

**Review by FelipeVelasquez1802 — APPROVED** [review]
# ✅Approved!

**Pull request merged**